WalletWasabi.Gui/Behaviors/BindSelectedTextBehavior.cs

```diff
@@ -44,11 +44,9 @@ protected override void OnAttached()
 		{
 			base.OnAttached();
 
-			AssociatedObject.GetObservable(TextBox.SelectionStartProperty).Merge(
-				AssociatedObject.GetObservable(TextBox.SelectionEndProperty)).Subscribe(_ =>
-				{
-					SelectedText = GetSelection();
-				});
+			AssociatedObject.GetObservable(TextBox.SelectionStartProperty)
+				.Merge(AssociatedObject.GetObservable(TextBox.SelectionEndProperty))
+				.Subscribe(_ => { SelectedText = GetSelection(); });
```

Comment:
It is against our code conventions. If there is a block it should be in separate lines with proper intentions.

Reply:
Actually he should remove the unnecessary brackets instead.

Reply:
Without the brackets it is fine.

Reply:
Fixed.